ruby/vampires.rb

```diff
@@ -0,0 +1,50 @@
+puts "How many employees will be processed?"
+x = gets.to_i
+x.times do 
+
+	puts "What is your name?"
+	user_name = gets.chomp
+
+	puts "How old are you?"
+	user_age = gets.to_i
+
+	puts "what year were you born?"
+	user_birthyear = gets.to_i
+	user_trueage = user_age + user_birthyear
+	
+	puts "Our company cafeteria serves bomb garlic bread. Should we order some for you? (y/n)"
+	garlic_bread = gets.chomp
+
+	puts "Would you like to enroll in the companys health insurance? (y/n)"
+	health_insur = gets.chomp
+	
+      if (user_name == "Drake Cula" || user_name == "Tu Fang") && user_trueage == "2016".to_i && garlic_bread == "y" && health_insur == "y"
```

Comment:
`"2016".to_i` is the same thing as writing `2016`, might as well use the integer version.